### PR TITLE
Update stack-monitoring.asciidoc

### PR DESCRIPTION
**Pull request by kunisen**
Per the PR below, add the minimum version to manual.

https://github.com/elastic/cloud-on-k8s/pull/4528
> enable stack logging for the ES container using the environment variable ES_LOG_STYLE=file
> Minimum supported Stack version is 7.14.0 (to benefit from ES_LOG_STYLE=file)

image

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? Yes
- Have you followed the [contributor guidelines](https://github.com/elastic/cloud-on-k8s/tree/master/CONTRIBUTING.md)? Yes
- If you submit code, is your pull request against master? We recommend pull requests against master. We will backport them as needed. => Not a code but a doc PR

### CODE REVIEW

File: docs/advanced-topics/stack-monitoring.asciidoc [hunk 1]
```diff
@@ -72,6 +72,8 @@ You can also enable Stack Monitoring on Elasticsearch only or on Kibana only. In
 
 == When to use it
 
+CAUTION: Minimum supported Stack version is 7.14.0.
```
Comment (thbkrkr) [inline docs/advanced-topics/stack-monitoring.asciidoc:75]: I think we prefer use a `NOTE` for this kind of information (example: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-install-helm.html#k8s-install-helm) but I would like to have @alaudazzi's opinion.
```suggestion
NOTE: The minimum supported Stack version is 7.14.0. 
```

Comment (kunisen) [reply docs/advanced-topics/stack-monitoring.asciidoc]: Thanks! Indeed NOTE is much better! ❤️

Comment (kunisen) [reply docs/advanced-topics/stack-monitoring.asciidoc]: I committed the suggestion for the time being but please let me know if we need to update more.

**Review by alaudazzi — APPROVED** [review]
LGTM

**Comment by pebrc** [comment]
I just realised that this should have targeted master and be back ported to 1.8. and 1.9 if necessary. 😞

**Pull request merged**